plugins/romega/refl_omega.ml

```diff
@@ -218,8 +217,8 @@ let display_omega_var i = Printf.sprintf "OV%d" i
    l'environnement initial contenant tout. Il faudra le réduire après
    calcul des variables utiles. *)
 
-let add_reified_atom t env =
-  try List.index0 Constr.equal t env.terms
+let add_reified_atom sigma t env =
+  try List.index0 (EConstr.eq_constr sigma) t env.terms
```

Comment:
@ppedrot I wonder why this naming anomaly...

Reply:
It's reusing a different scheme, because there are several competing notions of equality for terms. It should probably be uniformized with the other though.